templates/includes/pagination.html

```diff
@@ -0,0 +1,27 @@
+{% load cdh_tags %}
+
+<nav class="pagination" aria-label="Search pagination">
+  {% if page_obj.has_previous %}
+    <a rel="prev" href="?{% url_replace 'page' page_obj.previous_page_number %}" class="pagination__next-prev">
+      {% include 'includes/svg.html' with sprite="two-tone" svg="chevron-left" classes="pagination__icon pagination__icon--prev" %}
+      Previous
+    </a>
+  {% endif %}
+
+  {# just display all pages for now, no ellipses #}
+  {% for number in page_obj.paginator.page_range %}
+    <a href="?{% url_replace 'page' number %}" 
+      class="pagination__item {% if number == page_obj.number %}pagination__item--current{% endif %}"
+      >
+      {{ number }}
+    </a>
+  {% endfor %}
+
+  {% if page_obj.has_next %}
+    <a rel="next" href="?{% url_replace 'page' page_obj.next_page_number %}" class="pagination__next-prev">
+      Next
+      {% include 'includes/svg.html' with sprite="two-tone" svg="chevron-right" classes="pagination__icon pagination__icon--prev" %}
```

Comment:
does this want to be `next-prev` to match the other class name or perhaps drop the prev and next and just have `icon`?

Reply:
It's the only icon, but sure, I can make that change